src/main/java/org/betonquest/betonquest/compatibility/npcs/abstractnpc/NPCSupplierStandard.java

```diff
@@ -0,0 +1,26 @@
+package org.betonquest.betonquest.compatibility.npcs.abstractnpc;
+
+import org.betonquest.betonquest.exceptions.InstructionParseException;
+
+import java.util.function.Supplier;
+
+/**
+ * Supplies a NPC in an easy usable way in Events, Objectives, Conditions and Variables.
+ * <p>
+ * This is just a function which can throw an {@link InstructionParseException}.
+ */
+@FunctionalInterface
+public interface NPCSupplierStandard {
```

Comment:
I dont like the part `Standard` in the name

Reply:
I changed the name but have a kinda double now.